src/style.css

```
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form-input {
+  margin-top: 20px;
+}
+
+.input-big {
+  height: 70px;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.input-big::placeholder {
+  color: #3d4e61;
+}
+
+.input-small {
+  height: 42px;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+}
+
+.input-small::placeholder {
+  color: #3d4e61;
+}
```

you can combine these selectors, no need to repeat yourself
```suggestion
.input::placeholder {
  color: #3d4e61;
}

.input-small {
  height: 42px;
  padding-left: 33px;
  line-height: 19px;
  font-size: 14px;
}
```